composer.json

```
@@ -27,12 +27,12 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "composer-plugin-api": "^1.0",
+        "composer-plugin-api": "^1.0 || ^2.0",
         "laminas/laminas-component-installer": "^1.0 || ^2.1.2",
         "laminas/laminas-zendframework-bridge": "^1.0"
     },
     "require-dev": {
-        "composer/composer": "^1.4.1",
+        "composer/composer": "^1.4.1 || ^2.0@dev",
```

probably use : 

```suggestion
        "composer/composer": "^1.4.1 || ~2.0.0@dev || ^2.0",
```

to make consistent with https://github.com/mezzio/mezzio-skeleton/blob/ed34e5870cb0ed49baffd7e458b8d056593b29bc/composer.json#L53